_episodes/03-getting_started.md

```diff
@@ -82,25 +82,47 @@ Now we are ready to press `Create Project`.
 ## Play with the package
 
 Believe it or not, this package is ready to be installed.
-Just go to the upper right corner and press `Build > Install and Restart`.
+Just go to the upper right corner and press `Build > Install and Restart`. # TODO: not sure this has changed? In my console, I see `Build > Install`.
```

Comment:
I think this has changed, indeed! Perhaps good to have both here; some people may still have an older version (and the GIF is also from an earlier version).

Reply:
adjusted in new commit